src/Sulu/Bundle/MediaBundle/Resources/js/utils/MediaSelectionHelper/validateMediaType.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+export default function validateMediaType(name: ?string | number): boolean %checks {
+    return name === 'audio'
+        || name === 'document'
+        || name === 'image'
+        || name === 'video';
```

Comment:
That's connected to the comment above. If these types can be extended, we probably shouldn't check that here, because then this code would make the application throw an exception.